### ISSUE TITLE
Add feature to clone template

### ISSUE THREAD

**Issue by karuppiah7890** [freeCodeCamp/Mail-for-Good#128]
This might be useful to build upon other templates

**Comment by AndrewGHC** [comment]
This would definitely be useful as a feature. At present, one priority would be to allow the editing of templates, which is missing feature we're looking to implement.

**Comment by karuppiah7890** [comment]
Yes, true. In case there's an error or something. I did open an issue for it at #129
